src/main/java/core/basesyntax/service/impl/ValidateOperationImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ValidateOperation;
+
+public class ValidateOperationImpl implements ValidateOperation {
+    @Override
+    public void validate(Storage storage, Fruit fruit, int amount) {
+        if (!storage.getStorage().containsKey(fruit)) {
+            throw new RuntimeException("Incorrect operation!!! "
+                    + "This fruit does not have a balance!");
+        }
+    }
+}
```

Comment:
let's move this check into the strategies where it is used - and then we can remove this class and interface